Makefile

```diff
@@ -29,6 +29,16 @@ runmigrations:
 	env/bin/python wellregistry/manage.py migrate social_django
 	env/bin/python wellregistry/manage.py migrate registry
 
+runmigrations-containerized:
```

Comment:
Pretty a copy paste, but I'm expecting this to have a short-life time in this project.